### ISSUE TITLE
ObservableArray in Observable cannot work!

### ISSUE THREAD

**Issue by codeHui** [NativeScript/NativeScript#2682]
they all ObservableArray in Observable,but only the first can work!

===============
code here:
===============
first example

```
var Observable = require("data/observable").Observable;
import {HelloWorldModel} from "./modal";
var page;
var modal=new HelloWorldModel();
exports.loaded = function(args) {
    page = args.object;
    page.bindingContext = modal;
};
```

----------------
	modal.ts
```
import {ObservableArray} from 'data/observable-array';

export class HelloWorldModel extends Observable {
    list: ObservableArray<any>;
 
    constructor() {
        super();
        this.list = new ObservableArray([
        { name: "eggs" },
        { name: "bread" },
        { name: "cereal" }
    ]);
    }
}
```
===========
	second
```
var dialogsModule = require("ui/dialogs");
var Observable = require("data/observable").Observable;
var ObservableArray = require("data/observable-array").ObservableArray;
var page;
var ss=new ObservableArray([
        { name: "eggs" },
        { name: "bread" },
        { name: "cereal" }
    ]);

var pageData = new Observable({
    groceryList: ss
});

exports.loaded = function(args) {
    page = args.object;
    page.bindingContext = pageData;
};
```
===============
so, is 
```
	[export class HelloWorldModel extends Observable {
		list: ObservableArray<any>;]
```
have a big diff with
``` 
	[var pageData = new Observable({
	  groceryList: new ObservableArray()
	});]
```

**Comment by codeHui** [comment]
sorry,not good format;
be simple ,just this cannot work
var pageData = new Observable({
     groceryList: new ObservableArray()
});
and it seems I just put ObservableArray in another file and import it,then it can work,confused

**Comment by 3rror404** [comment]
I was having the same problem. Running `npm install tns-core-modules@2.1.0` to downgrade seems to have fixed the problem. Not sure if it will cause any other issues down the line.

**Comment by codeHui** [comment]
yes,this is an issue of 2.2.1

**Comment by NickIliev** [comment]
Hey @codeHui , @3rror404 

This is issue will be fixed with our next release (2.3.0) as discussed [here](https://github.com/NativeScript/NativeScript/issues/2645#issuecomment-244099315)
Possible temporary workaround (using @next) can be found  [here](https://github.com/NativeScript/NativeScript/issues/2603)

p.p. Closing this issue as duplicate - please track its development in the provided links above.
